nixos/modules/services/web-servers/garage.nix

```diff
@@ -75,7 +75,25 @@ in
       source = configFile;
     };
 
-    environment.systemPackages = [ cfg.package ]; # For administration
+    # For administration
+    environment.systemPackages = if cfg.environmentFile != null
+      then [(
+        # reads the env vars from `cfg.environmentFile` to the garage environment
+        # makes working with the cli quite a bit nicer
+        pkgs.writeScriptBin "garage" ''
```

Comment:
I think it's a good idea to name this wrapper in the let binding in the upper part of the file, and reference it here.

Reply:
sounds reasonable